cli/internal/helm/values.go

```diff
@@ -194,6 +194,8 @@ var ciliumVals = map[string]map[string]any{
 	},
 }
 
+var nodeSelectorForControlPlane = map[string]any{"node-role.kubernetes.io/control-plane": ""}
```

Comment:
```suggestion
var controlPlaneNodeSelector = map[string]any{"node-role.kubernetes.io/control-plane": ""}
```
Nit: goes a bit smoother and keeps in line with naming of `controlPlaneTolerations`